src/Web/Masa.Auth.Web.Sso/Pages/Protocol.razor

```diff
@@ -0,0 +1,14 @@
+﻿@page "/protocol"
+@inherits SsoCompontentBase
+@inject IWebHostEnvironment _environment
+@layout SimpleLayout
+
+<MCard Width="1000" Style="height: calc(100vh - 200px) !important;">
+    <MCardTitle Class="justify-center">
+        MASA Stack用户协议
+    </MCardTitle>
+    <MCardText Style="overflow-y: scroll;">
+        1. 特别提示
+        XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
Use DCC

Reply:
should sso client custom login setting,first provider page,next version improve function

Reply:
ok